charts/consul/templates/crd-routetimeoutfilters.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.9.2
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
I think it makes sense that this is "downgrading" based off of the version we should be installing with the makefile: https://github.com/hashicorp/consul-k8s/blob/main/Makefile#L186